qemu/tests/cfg/virtio_port_hotplug.cfg

```diff
@@ -43,7 +43,7 @@
                     repeat_times = 1
                 - with_bg_test:
                     Linux:
-                        filesize = 2000
+                        filesize = 400
```

Comment:
Could you please explain this change ? Thanks!

Reply:
2000 is too large for windows guest, sometimes it will cause date transfer timeout error, so change it, thanks.